src/engraving/layout/dev/chordlayout.cpp

```diff
@@ -2292,10 +2292,11 @@ void ChordLayout::layoutChords3(const MStyle& style, const std::vector<Chord*>&
     int prevSubtype = 0;
     int prevLine = std::numeric_limits<int>::min();
 
+    bool isTab = staff->isTabStaff(tick);
     for (int i = nNotes - 1; i >= 0; --i) {
         Note* note     = notes[i];
         Accidental* ac = note->accidental();
-        if (ac && ac->subtype() == prevSubtype && note->line() == prevLine) {
+        if (ac && (isTab || (ac->subtype() == prevSubtype && note->line() == prevLine))) { // TODO: probably this function shouldn't be called at all for tab staves
```

Comment:
// TODO: probably this function shouldn't be called at all for tab staves
 
I think we should try this :) It would be a great optimization for tab staves

Reply:
@RomanPudashkin done! I've exluded the TAB staff case earlier up the call stack, so it's not necessary to exclude it here

Reply:
Looks great! Please clarify for @zacjansheski what should be tested here. Thanks!